src/components/NavBar.js

```diff
@@ -16,21 +18,30 @@ function NavBar() {
           </span>
         </Link>
 
-      <div>
-        <button
-          type="button"
-          onClick={() => history.push('/signin')}
-        >
-          Log in
-        </button>
-        <button
-          type="button"
-          onClick={() => history.push('/signup')}
-        >
-          Registreren
-        </button>
-      </div>
-    </nav>
+        {isAuth ?
+            <button
+                type="button"
+                onClick={logout}
+            >
+              Log uit
+            </button>
+            :
+            <div>
```

Comment:
Als die div nergens anders voor gebruikt wordt, dan kun je beter een fragment gebruiken
```suggestion
            <>
```